client/my-sites/upgrades/domain-management/email-forwarding/email-forwarding-add-new.jsx

```diff
@@ -131,12 +157,14 @@ const EmailForwardingAddNew = React.createClass( {
 					<FormTextInputWithAffixes
 						disabled={ this.state.formSubmitting }
 						name="mailbox"
-						onChange={ this.handleFieldChange.bind( this, 'mailbox' ) }
+						onChange={ this.onChange }
 						onFocus={ this.handleFieldFocus.bind( this, 'Mailbox' ) }
+						isError={ ! isValidMailbox }
 						placeholder={ exampleEmailText }
 						type="text"
 						suffix={ '@' + this.props.selectedDomainName }
-						value={ this.state.mailbox } />
+						value={ mailbox } />
+					{ ! isValidMailbox ? <FormInputValidation text={ this.translate( 'Invalid mailbox' ) } isError={ true }/> : null }
```

Comment:
A slightly shorter way of writing this (and frequently used in Calypso) would be `! isValidMailbox && <FormInputValidation text={ this.translate( 'Invalid mailbox' ) } isError={ true }/>` - that's, like, 6 characters less! Wow, so optimal, much performance! :doge: